paramak/parametric_reactors/sparc_paper_2020.py

```diff
@@ -0,0 +1,279 @@
+
+import paramak
+
+
+class SparcFrom2020PaperDiagram(paramak.Reactor):
+    """Creates geometry of a simple SPARC reactor based on the published
+    diagram in Figure 4 of Overview of the SPARC tokamak. Journal of Plasma
+    Physics, 86(5), 865860502. doi:10.1017/S0022377820001257. Coordinates
+    extracted from the figure are not exact and therefore this model does not
+    perfectly represent the reactor.
+
+    Arguments:
+        rotation_angle (float): the angle of the sector that is desired.
+            Defaults to 360.0.
+    """
+
+    def __init__(
+        self,
+        rotation_angle=360.0,
+    ):
+
+        super().__init__([])
+
+        self.rotation_angle = rotation_angle
+
+    def create_solids(self):
```

Comment:
Maybe it's not a good idea to have a big create_solids method. Several options: splitting it in several methods, creating the solids "outside" of the class. What do you reckon @Shimwell ?

Reply:
Yep I agree and I think John does as well, this might be related to the discussion on adding common attributes to the classes. Such as .plasma .firstwall .blanket.